maestrowf/utils.py

```diff
@@ -349,3 +350,95 @@ def map_level(log_lvl):
             return logging.ERROR
         else:
             return logging.CRITICAL
+
+
+class ReconstructGlobalParameters:
```

Comment:
It may be worth renaming this to something more general like `MetadataParser` -- this class could be used as the starting point for other metadata parsing which contains methods for global parameter parsing. But I can easily envision this being extended to load in the metadata and search for workspaces, for example.